src/controller/discoverymgr/mocks/mocks_discoverymgr.go

```diff
@@ -1,140 +1,229 @@
-/*******************************************************************************
```

Comment:
Copyright statement should be attached properly (with the released year as 2020).

Reply:
Done